app/controllers/requests_controller.rb

```diff
@@ -8,4 +8,19 @@ def create
     @book = Book.find(params[:book_id])
     @request = @book.requests.create(user_id: current_user.id)
   end
+
+  def approve
+    request = Request.find(params[:id])
+    request.status = "approved"
```

Comment:
You can write this as `request.approved!` (enums creates that method for you), it saves, so there is no need for the line below